src/tests/Equinor.ProCoSys.IPO.WebApi.IntegrationTests/Invitations/InvitationsControllerTests.cs

```diff
@@ -116,15 +116,15 @@ public async Task SignPunchOut_AsSigner_ShouldSignPunchOut()
                     TestFactory.PlantWithAccess,
                     invitationToSignId,
                     participant.Person.Id,
-                    participant.RowVersion);
+                    participant.Person.RowVersion);
```

Comment:
In the current impl this is correct yes. 
But you see the symptoms of bad design. participant.Person here represent a participant in participant table, and not a person in person table. The original bug fixed (88613), was introduced because the programmer has mixed those entities. 

I have plans to flatten these participant DTOs, to get rid of the Person subclass, but this must be done carefully to not break frontend